tests/unit/plugins/module_utils/test_rest_client.py

```diff
@@ -64,6 +64,42 @@ def test_query_passing(self, client):
         )
 
 
+class TestTableListRecordsRaw:
+    def test_empty_response(self, client):
+        client.get.return_value = Response(
+            200, '{"result": []}', {"X-Total-Count": "0"}
+        )
+        t = rest_client.RestClient(client)
+
+        records = t.list_records_raw("my_table")
+
+        assert {"result": []} == records
+        client.get.assert_called_once_with(path="my_table", timeout=None)
+
+    def test_non_empty_response(self, client):
+        client.get.return_value = Response(
+            200, '{"result": [{"a": 3, "b": "sys_id"}]}', {"X-Total-Count": "1"}
+        )
+        t = rest_client.RestClient(client)
+
+        records = t.list_records_raw("my_table")
+
+        assert records == {"result": [{"a": 3, "b": "sys_id"}]}
+
+    def test_query_passing(self, client):
+        client.get.return_value = Response(
+            200, '{"result": []}', {"X-Total-Count": "0"}
+        )
+        t = rest_client.RestClient(client)
+
+        t.list_records_raw("my_table")
+
+        client.get.assert_called_once_with(
+            path="my_table",
+            timeout=None,
+        )
```

Comment:
This one is same as `test_empty_response`, only `list_records_raw` return value is not checked. We should drop the duplicated unittest.

Reply:
Thank you.